### DISCUSSION TITLE
Items with a red X are not planned, or not currently planned?

### DISCUSSION THREAD

**Discussion by porterwoodward** [awspring/spring-cloud-aws#404] [Q&A]
Hi -

just a quick clarification on the 3.0 roadmap and usage of the AWS 2.0 SDK.  Do the ❌'s indicate there is _no_ plan to support that feature of AWS, or that there is no current plan/task to do so?

I only ask as after having hacked together region support on the older AWS SDK with respect to controlling EC2 instances in a non-Spring context - I found that the 2.0 SDK had inbuilt support.  So as I push more for Spring-based and fewer home-rolled solutions, I was wondering if EC2 support for the newer Spring-Cloud-AWS was on the roadmap?

**Comment by maciejwalkowiak** [comment]
❌ means there is no plan but that does not mean that the plan cannot change. We realised that some of the integration we had do not bring enough value/are not used. What from the EC2 integration are you using specifically from 2.x version that you would like to see in 3.x?
